### ISSUE TITLE
optimizing interrupt routine

### ISSUE THREAD

**Issue by bkubicek** [bkubicek/Marlin#2]
for higher stepper rate

**Comment by bradleyf** [comment]
yes, since the physical machine is capable of much higher speeds than the motherboard/controller is capable of

**Comment by jv4779** [comment]
First step in a move takes 60us, acceleration/deceleration steps 30us, and cruising at 20us.

**Comment by bkubicek** [comment]
its in the works, evdz has sent me some new code that is able to do larger stepping rates. mergin the two branches will be however hard work.
